apps/site/src/components/canvas/View.tsx

```diff
@@ -1,28 +1,32 @@
 "use client";
 
+import { PropsWithChildren } from "react";
 import { forwardRef, useImperativeHandle, useRef } from "react";
-import { OrbitControls, View as ViewImpl } from "@react-three/drei";
+import { View as ViewImpl } from "@react-three/drei";
 import { Three } from "@/helpers/components/Three";
 
-const View = forwardRef(({ children, orbit, ...props }: any, ref) => {
-	const localRef = useRef<any>();
-	useImperativeHandle(ref, () => localRef.current);
+const View = forwardRef(
+	(
+		{ className, children }: PropsWithChildren<{ className: string }>,
+		ref,
+	) => {
+		const localRef = useRef<HTMLDivElement | null>(null);
+		useImperativeHandle(ref, () => localRef.current);
 
-	return (
-		<>
-			<div ref={localRef} {...props} />
-			{/* Sends View component thru tunnel to render on global Canvas in Scene.tsx */}
-			<Three>
-				{/* View component to render 3D view in a div (uses gl.scissor to cut viewport into segments) */}
-				<ViewImpl track={localRef}>
-					{children}
-					{/* drag controls to control camera in View*/}
-					{orbit && <OrbitControls />}
-				</ViewImpl>
-			</Three>
-		</>
-	);
-});
+		return (
+			<>
+				<div ref={localRef} className={className} />
+				{/* Sends View component thru tunnel to render on global Canvas in Scene.tsx */}
+				<Three>
+					{/* View component to render 3D view in a div (uses gl.scissor to cut viewport into segments) */}
+					{/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
+					{/* @ts-ignore */}
+					<ViewImpl track={localRef}>{children}</ViewImpl>
```

Comment:
Using a non-null assertion on the initial value would probably safer than completely ignoring types for the entire line. See [Typing with useRef in the React Three Fiber Tutorials](https://docs.pmnd.rs/react-three-fiber/tutorials/typescript#typing-with-useref).

Reply:
This is exactly what I was looking for, thank you!

Reply:
Or maybe the `ViewImpl` should be conditionally rendered when `localRef` is populated?

Reply:
Hmm, based on the article you sent, I think the non-null assertion operation is the best approach (and the simplest).